egs/wsj/s5/steps/nnet3/gru/decode.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
```

Comment:
Is there a need for a separate decode script ? Can you just use the one in LSTM. Actually with the latest changes made to support decoding of chain models we can use the tdnn decode script even for RNNs(LSTMs, BLSTMs, GRUs).